### DISCUSSION TITLE
Username allowing special characters

### DISCUSSION THREAD

**Discussion by andyondrya** [pelican/panel#731] [Ideas]
I think it might be better if we removed alphaNum from the user actions... mainly because you can use "-" or "_" in the username. If we did remove it, would there be anything breaking db wise?

**Comment by lancepioch** [comment]
Username rules were relaxed in [v1.0.0-beta26](https://github.com/pelican/panel/releases/tag/v1.0.0-beta26) via #1702 and loosened further in [v1.0.0-beta32](https://github.com/pelican/panel/releases/tag/v1.0.0-beta32) via #2150. Closing as resolved.
